lineax/_operator.py

```diff
@@ -538,6 +529,7 @@ def __init__(
         tags: Union[object, Iterable[object]] = (),
         closure_convert: bool = True,
         _has_aux: bool = False,  # TODO(kidger): remove, no longer used
+        use_jacrev: Union[bool, None] = None,
```

Comment:
I think I'd suggest a name that doesn't mention just `rev` in particular. E.g. something like
```python
jac: Optional[[Literal["fwd", "bwd"]]
```